src/Dapr.Client/Dapr.Client.csproj

```diff
@@ -22,7 +22,7 @@
   <ItemGroup>
     <PackageReference Include="Google.Protobuf" Version="3.15.0" />
     <PackageReference Include="Grpc.Net.Client" Version="2.32.0" />
```

Comment:
Should we be keeping the other gRPC libraries in parity with the one you updated?

Reply:
Shouldn't this project be updated to ?

![image](https://user-images.githubusercontent.com/1752811/184274432-4376a0ac-ef1a-4e67-898b-6625877a5eec.png)